### ISSUE TITLE
Chrome occupying the space of default hidden native scrollbar.

### ISSUE THREAD

**Issue by 7dhrumil** [KingSora/OverlayScrollbars#360]
Hello First of all Thank you so much for this awesome scrollbar,

I have tried to update the latest version of the OverlaysScrollbars and I found that It is adding some space at right side of the window for the chrome browser version 91.0.4472.7
Here I have attached the screenshots.

![image](https://user-images.githubusercontent.com/41570499/121038755-2998c400-c7ce-11eb-88a9-cc268200ccc0.png)

I think it is occupying the width of the native scrollbar.

If I am trying with to add the following css then its working properly.

```
::-webkit-scrollbar {
  width: 0px;
}
```
This is only happening in Chrome

`<div class="os-content" style="padding: 0px;height: 100%;width: 100%;border-right: 30px solid transparent;min-height: 31px;">`

I have noticed in the os-content div it is adding the `border-right: 30px` causing the issue

**Comment by KingSora** [comment]
Good day @7dhrumil 

OverlayScrollbars is doing a detection what type of scrollbars the browser is using. In your case it detected already overlaid scrollbars.. May I ask what is your Operating System? Do you set some kind of scrollbar styling to the whole page (css classes etc.) which could override the styles of the plugin?

**Comment by 7dhrumil** [comment]
I am using windows OS to browse my site. And No, I am not using any kind of scrollbar styling to the whole page. And I have also checked some of your comments and I have put OverlayScrollbars CSS on top of all other CSS. 

And This is only happening on the Server specifically for Chrome otherwise in my local is working fine. And Just curious about why it has an issue in Chrome Browser only?

**Comment by KingSora** [comment]
@7dhrumil do you mind sharing your page with me? If you don't want to post it publicly you can send me an email to the mail you can find on my profile. Since I have windows aswell, I should be able to reproduce it.

**Comment by 7dhrumil** [comment]
Hi @KingSora , I have sent you a mail where I have provided you credential of my site where ti is happening. Let me know if you need anything else.

**Comment by KingSora** [comment]
@7dhrumil I've taken a look and its kinda weird. I've found the reason of why this is happening:
As I explained in my first post, the plugin is doing some checking to analyze the environment it is operating in, and it needs CSS for that. The weird part is, that Firefox has loaded all the styles but Chrome has not.

Chrome:
![grafik](https://user-images.githubusercontent.com/12936317/121313024-46d0ae00-c906-11eb-8901-cb693cac3277.png)

Firefox:
![grafik](https://user-images.githubusercontent.com/12936317/121313385-9e6f1980-c906-11eb-82a4-0f02b20f670b.png)

I don't know why the CSS isn't loaded in chrome though.

**Comment by KingSora** [comment]
I've also found the reason behind the CSS loading.
In chrome the stylesheet is loaded like:
![grafik](https://user-images.githubusercontent.com/12936317/121314232-703e0980-c907-11eb-9c3a-30782b4d1e3b.png)
vs Firefox:
![grafik](https://user-images.githubusercontent.com/12936317/121314285-7d5af880-c907-11eb-9e28-8ae1b28a2d0f.png)

If I change the `media="print"` to `media="all"` in chrome, then its working and the styles are there.

**Comment by 7dhrumil** [comment]
Hi @KingSora , 
Thank you so much for your findings!!!.
After your finding, I have analyzed some more and Found that the Issue was due to the Angular Version 12. They have introduced something in the compiler that is adding some code from CSS or JS file inline in HTML which is critical and other CSS or js has to wait for that downloading and they are not taking whole CSS or js but the bunch of the code which is necessary primarily and then they are inserting another remaining chunk I think which is breaking and creating issue to identify overlay scrollbars js to analze.

Here is the link if you are interested to check that fact.

https://stackoverflow.com/a/67582075/13533436

But again thank you so much for your time and for this awesome component...
